### PR TITLE
Defect/kitosudv 2098 task 2100

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/app/utility/kendoGridLauncher.ts [hunk 1]
```diff
@@ -108,30 +106,31 @@ module Kitos.Utility.KendoGrid {
...
+        build(): IExtendedKendoGridColumn<TDataSource> {
             this.checkRequiredField("title", this.title);
             this.checkRequiredField("id", this.id);
+            this.checkRequiredField("dataSourceName", this.dataSourceName);
             this.checkRequiredField("excelOutput", this.excelOutput);
 
             return {
+                field: `${this.dataSourceName}_${new Date().getTime()}`, //Make the persistid random every time the grid is built (an error in the export process should not "stick")
                 title: this.title,
+                attributes: [],
```
Comment (JacobMalling) [inline Presentation.Web/app/utility/kendoGridLauncher.ts:124]: Kan der sættes data på denne?

File: Presentation.Web/app/utility/kendoGridLauncher.ts [hunk 2]
```diff
@@ -108,30 +106,31 @@ module Kitos.Utility.KendoGrid {
...
 
-        build(): IKendoGridExcelOnlyColumn<TDataSource> {
+        build(): IExtendedKendoGridColumn<TDataSource> {
             this.checkRequiredField("title", this.title);
             this.checkRequiredField("id", this.id);
+            this.checkRequiredField("dataSourceName", this.dataSourceName);
             this.checkRequiredField("excelOutput", this.excelOutput);
 
             return {
+                field: `${this.dataSourceName}_${new Date().getTime()}`, //Make the persistid random every time the grid is built (an error in the export process should not "stick")
```
Comment (JacobMalling) [inline Presentation.Web/app/utility/kendoGridLauncher.ts:122]: Det er da field og ikke persistId du sætter her?

File: Presentation.Web/app/utility/kendoGridLauncher.ts [hunk 3]
```diff
@@ -108,30 +106,31 @@ module Kitos.Utility.KendoGrid {
...
                 title: this.title,
+                attributes: [],
                 width: this.standardWidth,
-                id: this.id,
-                template: (dataItem => this.excelOutput(dataItem)),
-                dependOnColumnId: this.dependOnColumnId
-            } as IKendoGridExcelOnlyColumn<TDataSource>;
+                persistId: this.id,
```
Comment (JacobMalling) [inline Presentation.Web/app/utility/kendoGridLauncher.ts:126]: Det er da på denne vi skal have det der new Date().getTime() tilføjet

**Review by JacobMalling — CHANGES_REQUESTED** [review]
Det ligner da at der lige skal rettes lidt i kendoGridLauncher

File: Presentation.Web/app/utility/kendoGridLauncher.ts (hunk 2 above)
Comment (mrjsawdk) [reply Presentation.Web/app/utility/kendoGridLauncher.ts:122]: yes

File: Presentation.Web/app/utility/kendoGridLauncher.ts (hunk 1 above)
Comment (mrjsawdk) [reply Presentation.Web/app/utility/kendoGridLauncher.ts:124]: ikke på excel fields

File: Presentation.Web/app/utility/kendoGridLauncher.ts (hunk 3 above)
Comment (mrjsawdk) [reply Presentation.Web/app/utility/kendoGridLauncher.ts:126]: korrekt

**Review by JacobMalling — APPROVED** [review]
Det ser bedre ud 👍

**Pull request merged**